### ISSUE TITLE
All SVG elements should have focusable="false" attribute

### ISSUE THREAD

**Issue by danchamp** [localgovdrupal/localgov_base#110]
[Accessibility audit: issue 59](https://docs.google.com/spreadsheets/d/1t8rRIQe_1kogxPBZDKcsbf8YxtEfC9iGJvidkagNSXY/edit#gid=1779042480&range=E18)
